src/components/suggest_item/suggest_item.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { EuiIcon, IconPropType } from '../icon';
+
+const colorToClassNameMap = {
+  tint0: 'euiSuggestItem__type--tint0',
+  tint1: 'euiSuggestItem__type--tint1',
+  tint2: 'euiSuggestItem__type--tint2',
+  tint3: 'euiSuggestItem__type--tint3',
+  tint4: 'euiSuggestItem__type--tint4',
+  tint5: 'euiSuggestItem__type--tint5',
+  tint6: 'euiSuggestItem__type--tint6',
+  tint7: 'euiSuggestItem__type--tint7',
+  tint8: 'euiSuggestItem__type--tint8',
+  tint9: 'euiSuggestItem__type--tint9',
+};
+
+export const COLORS = Object.keys(colorToClassNameMap);
+
+const labelDisplayToClassMap = {
+  fixed: 'euiSuggestItem__labelDisplay--fixed',
+  expand: 'euiSuggestItem__labelDisplay--expand',
+};
+
+export const DISPLAYS = Object.keys(labelDisplayToClassMap);
+
+export const EuiSuggestItem = ({
+  className,
+  label,
+  type,
+  labelDisplay,
+  description,
+  ...rest
+}) => {
+  const classes = classNames('euiSuggestItem', className);
+
+  let optionalColorClass = '';
+  let labelDisplayClass = labelDisplayToClassMap[labelDisplay];
```

Comment:
_**optional**_

You can also use the `classNames()` function here to help concatenate the classes like:

```suggestion
  let labelDisplayClass = classNames(
    'euiSuggestItem__label', 
    labelDisplayToClassMap[labelDisplay],
    {
      'euiSuggestItem__labelDisplay--expand': !description,
    }
  );
```

It's basically just shorthand for what you were already doing. I'm totally fine if you keep it the way you have it. Just a suggestion.